time stamp
bool velocityMode
string[] id
float64[] position_cmd
float64[] velocity_cmd


